tools/install-powershell.ps1

```diff
@@ -99,10 +113,17 @@ try {
 
         $package = Find-Package -Source powershell-core-daily -AllowPrereleaseVersions -Name $packageName
         Write-Verbose "Daily package found. Name: $packageName; Version: $($package.Version)" -Verbose
+
         Install-Package -InputObject $package -Destination $tempDir -ExcludeVersion > $null
+        Remove-Destination $Destination
 
         $contentPath = [System.IO.Path]::Combine($tempDir, $packageName, "content")
-        Copy-Item -Path $contentPath\* -Destination $Destination -Recurse -Force
+        if (Test-Path $Destination) {
+            Write-Verbose "Copying files" -Verbose
+            Copy-Item -Recurse -Path "$contentPath\*" -Destination $Destination -ErrorAction Ignore
```

Comment:
We shouldn't specify `-ErrorAction Ignore`, if it fails to copy a file then the installation fails. And maybe add `-Force`.

Reply:
The failure is due to it copying folders, I can change this to filter to just files and letting it fail